netbox/ipam/tests/test_filtersets.py

```diff
@@ -1724,6 +1724,15 @@ def test_contains_vid(self):
         params = {'contains_vid': 1}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 8)
 
+    def test_contains_vid_upper_bound_exclusive(self):
+        # 11 is NOT in [1,11)
+        params = {'contains_vid': 12}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+
+    def test_contains_vid_miss(self):
+        params = {'contains_vid': 4095}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 0)
+
```

Comment:
IMO we should wrap these into `test_contains_vid()` above as they all test the same filter.